### PR TITLE
Remove Photo Caption related helpers

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request removes helpers and tests for captions throughout the codebase. We are removing these since we are no longer requiring a caption in the photo submission via the website and we are going to be removing the caption request from our sms flow. (happening in #558)

### How should this be reviewed?

👀  - let me know if anything seems like it should be left in or we need to make more clarifications. I'd recommend going commit by commit. A lot of the "changes" here are linter cleanup.

### Any background context you want to provide?

No need to keep references or helpers for something we are no longer asking for!

### Relevant tickets

References [Pivotal #175786378](https://www.pivotaltracker.com/n/projects/2401401/stories/175786378).

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tests added/updated for new features/bug fixes.
- [ ] ENV variables added/updated for new features/bug fixes.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

File: config/lib/helpers/template.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ const templatesMap = {
     invalidAskSignupResponse: 'invalidAskSignupResponse',
   },
   topicTemplates: {
-    askCaption: 'askCaption',
```
Comment (katiecrane) [inline config/lib/helpers/template.js:22]: Is this only used for photo posts? I'm guessing the `askText` is what we use for text posts?

Comment (lindsayrmaher) [reply config/lib/helpers/template.js:22]: I believe that's correct! I only saw askCaption being used in photoPost related spots and askText with textPosts https://github.com/DoSomething/gambit/blob/13039f30c58d990608ec1b5ea87e7b205ee0fe91/config/lib/helpers/topic.js#L38-L38

**Comment by mendelB** [comment]
💥

File: config/lib/graphql.js [hunk 2]
```diff
@@ -214,11 +214,9 @@ const fetchTopicById = `
       }
       ... on PhotoPostTopic {
         ...photoPostCampaign
-        askCaption
```
Comment (mendelB) [inline config/lib/graphql.js:217]: Do these caption fields need to be deleted from Contentful / GraphQL as well? 🤔

Comment (lindsayrmaher) [reply config/lib/graphql.js:217]: that makes sense to me!

**Pull request merged**